src/main/java/gnu/io/CommPortIdentifier.java

```diff
@@ -115,7 +114,7 @@
 		} 
 		catch (Throwable e) 
 		{
-			log.error("Exception thrown while loading gnu.io.RXTXCommDriver", e);
+			log.error("Failed to load gnu.io.RXTXCommDriver: {}", e.getMessage());
```

Comment:
Logging the exception directly will print its stack trace, right? Or are you intentionally suppressing that behaviour?

```suggestion
			log.error("Failed to load gnu.io.RXTXCommDriver:", e);
```

Reply:
I log stack traces only if the method can be called from different places, i.e. if you really need the stack trace when investigating an error to identify the caller. But I'm fine if we log the stack trace here.

Reply:
Ah, right, that makes sense. Although I wonder how many people use a stack trace in a situation like this to quickly identify where _their_ code calls into the library. But let's keep it like this for now. I noticed another case of logging the message but not the stack trace, and it's probably the same situation as this.